docs/integrations.rst

```diff
@@ -104,3 +104,15 @@ To get it working you need to export the following environment variable:
 
 ``GOOGLE_ANALYTICS_TRACKING_ID``
   Your page's Google “Tracking ID.“
+
+
+Full text search with Elasticsearch
+-----------------------------------
+
+You can use optional `Elasticsearch <https://www.elastic.co/products/elasticsearch>`_ integration.
+To get it working you need to export following environment variable:
+
+``ELASTICSEARCH_URL``
```

Comment:
I'd drop the variable from here as there's more than one and there is no point in duplicating the descriptions.